common/changes/office-ui-fabric-react/nidurak-datePickerEmpty_2018-11-01-22-38.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "propaget empty input in DatePicker to onSelectedDate even when input required",
```

Comment:
typo

Reply:
Also, can you start the changefile comment with the affected component. These comments get listed in our release notes.